src/main/java/ua/dymohlo/multithreadinghwpart2/Counter.java

```diff
@@ -0,0 +1,16 @@
+package ua.dymohlo.multithreadinghwpart2;
+
+public class Counter implements Runnable{
```

Comment:
N; Here the same thing. While I dont check the same class I dont understand what this class does by naming. So it would nice if you call this:
IncrementRunnable